### PR TITLE
WIP How it works slider

### PR DESCRIPTION
**Pull request by priyanka9634**
WIP how-it-works slider https://admint.io/#/home

### CODE REVIEW

File: common/fonts.ftd [hunk 1]
```diff
@@ -226,6 +226,24 @@ desktop: $size-23-desktop
...
+-- ftd.type size-24-regular-desktop:
+line-height.px: 31
+size.px: 23
+letter-spacing.px: 0
+font-family: $nunito-assets.fonts.Nunito
+weight: 700
+
+-- ftd.responsive-type size-24-regular:
+desktop: $size-24-regular-desktop
+mobile: $size-24-regular-mobile
```
Comment (Arpita-Jaiswal) [inline common/fonts.ftd:246]:
```suggestion
-- ftd.type size-24-regular-desktop:
line-height.px: 31
size.px: 23
letter-spacing.px: 0
weight: 700
font-family: $nunito-assets.fonts.Nunito


-- ftd.responsive-type size-24-regular:
desktop: $size-24-regular-desktop
```

File: lib.ftd [hunk 2]
```diff
@@ -1895,12 +2093,42 @@ region: h2
 margin-bottom.px: 28
 
 -- ftd.text:
+if: { ftd.device != "mobile" }
```
Comment (Arpita-Jaiswal) [inline lib.ftd:2096]:
```suggestion
```

Unnecessary condition as this condition is already given to parent

File: lib.ftd [hunk 3]
```diff
@@ -1946,74 +2230,190 @@ padding-horizontal.px: 12
...
+height.fixed.px: 120
+
+-- ftd.text: $works-cell-desktop.bottom-cell
 role: $inherited.types.blockquote
 color: $inherited.colors.background.step-1
 padding-vertical.px: 13
 padding-horizontal.px: 17
-width.fixed.percent: 100
+
+-- end: ftd.row
```
Comment (Arpita-Jaiswal) [inline lib.ftd:2283]: Why row is added?

Can we do this instead?

```ftd
-- ftd.text: $works-cell-desktop.bottom-cell
role: $inherited.types.blockquote
color: $inherited.colors.background.step-1
padding-vertical.px: 13
padding-horizontal.px: 17
width: fill-container
height.fixed.px: 120
```

File: lib.ftd [hunk 4]
```diff
@@ -1946,74 +2230,190 @@ padding-horizontal.px: 12
...
+if: { works-cell-mobile.left-mob-src == NULL }
+src: $works-cell-mobile.left-src
+width.fixed.percent: 27
+height: auto
 
+-- ftd.image:
+if: { works-cell-mobile.left-mob-src != NULL }
+src: $works-cell-mobile.left-mob-src
+width.fixed.percent: 27
+height: auto
```
Comment (Arpita-Jaiswal) [inline lib.ftd:2340]:
```suggestion
-- ftd.image:
src if { works-cell-mobile.left-mob-src != NULL }: $works-cell-mobile.left-mob-src
src: $works-cell-mobile.left-src
width.fixed.percent: 27
height: auto
```

File: lib.ftd [hunk 5]
```diff
@@ -1946,74 +2230,190 @@ padding-horizontal.px: 12
...
+if: { works-cell-mobile.right-mob-src == NULL }
+src: $works-cell-mobile.right-src
+width.fixed.percent: 27
+height: auto
+
+-- ftd.image:
+if: { works-cell-mobile.right-mob-src != NULL }
+src: $works-cell-mobile.right-mob-src
+width.fixed.percent: 27
+height: auto
```
Comment (Arpita-Jaiswal) [inline lib.ftd:2352]:
```suggestion
-- ftd.image:
src: if { works-cell-mobile.right-mob-src != NULL }: $works-cell-mobile.right-mob-src
src: $works-cell-mobile.right-src
width.fixed.percent: 27
height: auto
```

File: lib.ftd [hunk 6]
```diff
@@ -1946,74 +2230,190 @@ padding-horizontal.px: 12
...
+color: $inherited.colors.background.step-1
+
+-- end: ftd.row
+
+-- ftd.text: $works-cell-mobile.title
+role: $fonts.size-19
+color: $inherited.colors.background.step-1
+width: fill-container
+
+-- end: ftd.row
```
Comment (Arpita-Jaiswal) [inline lib.ftd:2382]: Given attribute which applies to child like `color` and `role` etc, should be given to parent only. Not to individual child. 

```suggestion
-- ftd.row:
background.solid: $inherited.colors.border
padding-vertical.px: 7
padding-horizontal.px: 12
width: fill-container
spacing.fixed.px: 6
role: $fonts.size-19
color: $inherited.colors.background.step-1

-- ftd.row:
spacing.fixed.px: 2

-- ftd.integer: $works-cell-mobile.number

-- ftd.text: .

-- end: ftd.row

-- ftd.text: $works-cell-mobile.title
width: fill-container

-- end: ftd.row
```

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

File: lib.ftd [hunk 7]
```diff
@@ -3625,3 +4025,338 @@ align-self: start
...
+
+-- ftd.text: Step
+role: $fonts.size-19
+color: $inherited.colors.custom.four
+
+-- ftd.integer: $works-cell-desktop.number
+role: $fonts.size-19
+color: $inherited.colors.custom.four
+
+-- end: ftd.row
```
Comment (Arpita-Jaiswal) [inline lib.ftd:4192]:
```suggestion
-- ftd.row:
width: fill-container
align-content: center
spacing.fixed.px: 6
role: $fonts.size-19
color: $inherited.colors.custom.four

-- ftd.text: Step
-- ftd.integer: $works-cell-desktop.number

-- end: ftd.row
```

File: lib.ftd [hunk 8]
```diff
@@ -3625,3 +4025,338 @@ align-self: start
...
+width: fill-container
+height.fixed.px: 120
+
+-- ftd.text: $works-cell-desktop.top-cell
+role: $inherited.types.blockquote
+color: $inherited.colors.custom.one
+padding-vertical.px: 13
+padding-horizontal.px: 17
+
+-- end: ftd.row
```
Comment (Arpita-Jaiswal) [inline lib.ftd:4218]:
```suggestion
-- ftd.text: $works-cell-desktop.top-cell
role: $inherited.types.blockquote
color: $inherited.colors.custom.one
padding-vertical.px: 13
padding-horizontal.px: 17
width: fill-container
height.fixed.px: 120
text-align: center
```

File: lib.ftd [hunk 9]
```diff
@@ -3625,3 +4025,338 @@ align-self: start
...
+width: fill-container
+height.fixed.px: 120
+
+-- ftd.text: $works-cell-desktop.bottom-cell
+role: $inherited.types.blockquote
+color: $inherited.colors.background.step-1
+padding-vertical.px: 13
+padding-horizontal.px: 17
+
+-- end: ftd.row
```
Comment (Arpita-Jaiswal) [inline lib.ftd:4231]:
```suggestion
-- ftd.text: $works-cell-desktop.bottom-cell
role: $inherited.types.blockquote
color: $inherited.colors.background.step-1
padding-vertical.px: 13
padding-horizontal.px: 17
text-align: center
width: fill-container
height.fixed.px: 120
```

File: lib.ftd [hunk 10]
```diff
@@ -3625,3 +4025,338 @@ align-self: start
...
+if: { works-cell-mobile.left-mob-src == NULL }
+src: $works-cell-mobile.left-src
+width.fixed.percent: 27
+height: auto
+
+-- ftd.image:
+if: { works-cell-mobile.left-mob-src != NULL }
+src: $works-cell-mobile.left-mob-src
+width.fixed.percent: 27
+height: auto
```
Comment (Arpita-Jaiswal) [inline lib.ftd:4288]:
```suggestion
-- ftd.image:
src if { works-cell-mobile.left-mob-src != NULL }: $works-cell-mobile.left-mob-src
src: $works-cell-mobile.left-src
width.fixed.percent: 27
height: auto
```

File: lib.ftd [hunk 11]
```diff
@@ -3625,3 +4025,338 @@ align-self: start
...
+if: { works-cell-mobile.right-mob-src == NULL }
+src: $works-cell-mobile.right-src
+width.fixed.percent: 27
+height: auto
+
+-- ftd.image:
+if: { works-cell-mobile.right-mob-src != NULL }
+src: $works-cell-mobile.right-mob-src
+width.fixed.percent: 27
+height: auto
```
Comment (Arpita-Jaiswal) [inline lib.ftd:4300]:
```suggestion
-- ftd.image:
src if { works-cell-mobile.right-mob-src != NULL }: $works-cell-mobile.right-mob-src
src: $works-cell-mobile.right-src
width.fixed.percent: 27
height: auto
```

File: lib.ftd [hunk 12]
```diff
@@ -3625,3 +4025,338 @@ align-self: start
...
+color: $inherited.colors.background.step-1
+
+-- end: ftd.row
+
+-- ftd.text: $works-cell-mobile.title
+role: $fonts.size-19
+color: $inherited.colors.background.step-1
+width: fill-container
+
+-- end: ftd.row
```
Comment (Arpita-Jaiswal) [inline lib.ftd:4329]:
```suggestion
-- ftd.row:
background.solid: $inherited.colors.border
padding-vertical.px: 7
padding-horizontal.px: 12
width: fill-container
spacing.fixed.px: 6
role: $fonts.size-19
color: $inherited.colors.background.step-1

-- ftd.row:
spacing.fixed.px: 2

-- ftd.integer: $works-cell-mobile.number
-- ftd.text: .

-- end: ftd.row

-- ftd.text: $works-cell-mobile.title
width: fill-container

-- end: ftd.row
```

File: lib.ftd [hunk 13]
```diff
@@ -3625,3 +4025,338 @@ align-self: start
...
+margin-bottom.px: 10
+
+-- end: ftd.row
+
+-- ftd.text: $works-cell-mobile.bottom-cell
+role: $inherited.types.blockquote
+color: $inherited.colors.text-strong
+text-align: center
+width: fill-container
+
```
Comment (Arpita-Jaiswal) [inline lib.ftd:4357]:
```suggestion
-- ftd.column:
width: fill-container
padding-vertical.px: 17
padding-horizontal.px: 10
role: $inherited.types.blockquote
color: $inherited.colors.text-strong

-- ftd.text: $works-cell-mobile.top-cell
text-align: center
width: fill-container

-- ftd.row:
width.fixed.percent: 70
align-self: center
height.fixed.px: 1
background.solid: $inherited.colors.border
margin-top.px: 10
margin-bottom.px: 10

-- end: ftd.row

-- ftd.text: $works-cell-mobile.bottom-cell
text-align: center
width: fill-container

```

File: lib.ftd [hunk 14]
```diff
@@ -3625,3 +4025,338 @@ align-self: start
...
+-- ftd.row:
+width.fixed.percent: 70
+align-self: center
+height.fixed.px: 1
+background.solid: $inherited.colors.border
```
Comment (Arpita-Jaiswal) [inline lib.ftd:4346]: Why is `border` color used for `background`?

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

File: lib.ftd (hunk 14 above)
Comment (priyanka9634) [reply lib.ftd:4300]: because we need a color value that color code is matched with border color

Comment (gsalunke) [reply lib.ftd:4300]: @Arpita-Jaiswal resolving this - as this color scheme needed some more colors where some colors added in different color types.

**Pull request closed without merging**